### PR TITLE
implemented the command line parser to read the stock ticker which is…

### PR DESCRIPTION
**Pull request by iniadeniyi**
… passed as a command line argument. Also implemented the functionalities to validate the stock ticker

### CODE REVIEW

File: stock_monitor/target/.rustc_info.json
Comment (dheadrick1618) [inline stock_monitor/target/.rustc_info.json:1]: Please remove this file. Not sure why the .gitignore isn't working with this. Ill add  this explicitly to the gitignore . Same is happening with me.

**Review by dheadrick1618 — CHANGES_REQUESTED** [review]
Ron taught me a while back that you should only ever explicitly include files when committing. Never use ' git add .  '  It prevents things like the info.json file you have to get tossed in, when its just a random build file. 

Other than that looks good!

**Comment by iniadeniyi** [comment]
I believe it's fixed now.

File: stock_monitor/src/command_line_parser.rs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+fn check_stock_ticker_length(ticker: &str) -> bool {
+    ticker.len() >= 3 && ticker.len() <= 5
+}
+
+fn check_stock_ticker_alphabetic(ticker: &str) -> bool {
+    ticker.chars().all(|c: char| c.is_alphabetic())
+}
+
+pub fn check_stock_ticker(ticker: String) -> bool {
```
Comment (Koss24) [inline stock_monitor/src/command_line_parser.rs:27]: Do we need a String as parameter here we use a string pointer every else. It doesn't look like the String is being modified here so I don't think we need it be mutable and a pointer is fine

**Review by Koss24 — CHANGES_REQUESTED** [review]

**Pull request merged**